hazelcast/src/main/java/com/hazelcast/internal/serialization/impl/compact/CompactUtil.java

```diff
@@ -30,18 +30,20 @@ private CompactUtil() {
 
     @Nonnull
     static HazelcastSerializationException exceptionForUnexpectedNullValue(@Nonnull String fieldName,
-                                                                           @Nonnull String methodSuffix) {
+                                                                           @Nonnull String methodSuffix,
+                                                                           @Nonnull String methodPrefix) {
```

Comment:
I believe it would read better if the prefix comes before the suffix in the parameter list